netmaster/master/network.go

```diff
@@ -163,6 +164,57 @@ func deleteDockNet(networkName string) error {
 	return nil
 }
 
+func detachServiceContainer(networkName, tenantName string) error {
```

Comment:
This is something we need to think about as we'll have more such co-processing containers like ACI-gateway and skydns. Current thinking for skydns is to let schedulers(E.g. Swarm) run it anywhere in the cluster where resources are available. ACI-gateway might have slightly different requirements around locality with active-netmaster.